src/Tasks/Common/Resources/Strings.resx

```diff
@@ -758,4 +758,12 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1163: Cannot emit symbols when publishing for .NET 5 with Crossgen2 using composite mode.</value>
     <comment>{StrBegin="NETSDK1163: "}</comment>
   </data>
-</root>
+  <data name="TrimmingWindowsFormsRequiresCom" xml:space="preserve">
+    <value>NETSDK1164: Trimming Windows Forms is dangerous and requires 'BuiltInComSupport' to be set to 'true'.</value>
+    <comment>{StrBegin="NETSDK1164: "}</comment>
+  </data>
+  <data name="TrimmingWpfIsDangerous" xml:space="preserve">
+    <value>NETSDK1165: Trimming WPF is dangerous and might have issues at runtime.</value>
```

Comment:
Same feedback as for the above. I'd avoid the "dangerous" terminology and focus on trim-compatibility and that the combination is not supported/recommended.